src/hooks/useCalculatedColumns.ts

```diff
@@ -81,7 +81,7 @@ export function useCalculatedColumns<R, SR>({
       }
 
       return column;
-    });
+    }).filter(Boolean);
```

Comment:
This array never has falsy values, so this does nothing.

Reply:
Sorry, it's my problem, I can handle the source data;
Actually, it's like this, for example Array.fill(10).map(item=>({. .item})), it will still return [empty * 10];
It's possible to filter out the damn troubleshooting time here, especially reading data from xlsx. 
Thank you for your reply!

Reply:
This should be the responsibility of the parent components, I don't think RDG should have to check for holey arrays.

Reply:
👌，Thank you for your reply